owltorrent-network/src/main/java/edu/rice/owltorrent/network/ClientHandler.java

```diff
@@ -19,6 +17,8 @@
   private DataOutputStream out;
   private DataInputStream in;
 
+  private TorrentManager torrentManager;
```

Comment:
why is this a field? it should get a new torrent manager every time it recieves a new handshake right?

Reply:
Yes, and it's pretty bad practice. I just didn't want to return a tuple in the verifyHandshake method, and the need for TorrentManager in ClientHandler's run should be temporary -- for now Socket Connector will need TorrentManager, but later on we should find a way to make the two integrate better. 
I can change it into retrieving the TorrentManager a second time in run if you want.

Reply:
just add a TODO for now it's fine